cli/lakeflags/state.go

```diff
@@ -3,13 +3,25 @@ package lakeflags
 import (
 	"fmt"
 	"os"
+	"path/filepath"
 	"strings"
 )
 
-const headFile = ".zed_head"
+const headFile = ".ZED_HEAD"
```

Comment:
I much prefer `.zed_head` to `.ZED_HEAD`. There's no need to shout.

Reply:
Unrelated: There will one day be "Zed Head" t-shirts.

Reply:
...and perhaps even more appropriate, hats!